code/game/area/ss13_areas/medical_areas.dm

```diff
@@ -39,6 +39,16 @@
 	icon_state = "patients"
 	sound_environment = SOUND_AREA_SMALL_SOFTFLOOR
 
+/area/station/medical/patients_rooms1
+	name = "\improper Patient Room"
+	icon_state = "patients"
+	sound_environment = SOUND_AREA_SMALL_SOFTFLOOR
+
+/area/station/medical/patients_rooms2
+	name = "\improper Patient's Rooms"
```

Comment:
You forgot to tweak the other area's name as well

Reply:
fixed